src/mixins/filing-template-mixin.ts

```diff
@@ -428,9 +416,6 @@ export default class FilingTemplateMixin extends DateMixin {
       }
     }
 
-    // delete invalid tax id (see ticket 15122)
-    delete filing.business.taxId
```

Comment:
I tested this filing (same as original bug ticket) with a BN15 and it was accepted and completed correctly.